youtrack_sdk/client.py

```diff
@@ -133,6 +133,20 @@ def _post(
     def _delete(self, *, url: str) -> Optional[dict]:
         return self._send_request(method=HTTPMethod.DELETE, url=url)
 
+    def get_absolute_url(self, *, path: str) -> str:
+        return f"{self._base_url}{path}"
+
+    def issue_exists(self, *, issue_id: str) -> bool:
```

Comment:
So why in the end you decided against a generic version parameterised by type?

Reply:
Because if you want to check for the existence of a comment or attachment, you need to pass `issue_id` and `object_id` instead of just `object_id`. Also I didn't find the real reason for checking anything else except the issue.

Reply:
Maybe we can then add a helper `exists` to the `helpers`:

```python
def exists[T](getter: Callable[..., T], *args, **kwargs) -> Optional[T]:
    try:
        obj = getter(*args, **kwargs)
    except YouTrackNotFound:
        return None
    else:
        return obj


exists(client.get_issue, "MBO-123")

```

Alternative, but I like it less even though it has code completion advantages:

```python
exists(lambda: client.get_issue("MBO-123"))
```